### ISSUE TITLE
My Winget not working!!

### ISSUE THREAD

**Issue by NVT223** [ChrisTitusTech/winutil#1210]
**Describe the bug**
Package agreements were not agreed to.

**To Reproduce**
1. Go to 'irm christitus.com/win | iex'
2. Click on 'Installed'
3. Scroll down to bottom

**Expected behavior**
Winget not working

**Screenshots**
![Screenshot (7)](https://github.com/ChrisTitusTech/winutil/assets/114287428/ac83c016-76a0-4ee9-a855-9e010e65e865)

![Screenshot (9)](https://github.com/ChrisTitusTech/winutil/assets/114287428/099c53e9-38f2-4ba6-8390-5ea2247291ca)

**Additional context**
Anyone could help me that be great!!

**Comment by munkk01** [comment]
I noticed the issue you opened under the winget repo:
https://github.com/microsoft/winget-cli/issues/3826

Have you taken a look at this issue under the same repo with the attempted/suggested fixes?
https://github.com/microsoft/winget-cli/issues/566

Based on the text you provided under the winget-cli issue, it's hard to say this is a direct issue with WinUtil here.

**Comment by NVT223** [comment]
> I noticed the issue you opened under the winget repo: [microsoft/winget-cli#3826](https://github.com/microsoft/winget-cli/issues/3826)
> 
> Have you taken a look at this issue under the same repo with the attempted/suggested fixes? [microsoft/winget-cli#566](https://github.com/microsoft/winget-cli/issues/566)
> 
> Based on the text you provided under the winget-cli issue, it's hard to say this is a direct issue with WinUtil here.

yes im sorry but idk how to fix that. Im google a lot of solutions but impossible to fix. If u know how to fix can u show me how. Im appreciated for your help. Thanks

**Comment by ChrisTitusTech** [comment]
> > I noticed the issue you opened under the winget repo: [microsoft/winget-cli#3826](https://github.com/microsoft/winget-cli/issues/3826)
> > 
> > Have you taken a look at this issue under the same repo with the attempted/suggested fixes? [microsoft/winget-cli#566](https://github.com/microsoft/winget-cli/issues/566)
> > 
> > Based on the text you provided under the winget-cli issue, it's hard to say this is a direct issue with WinUtil here.
> 
> yes im sorry but idk how to fix that. Im google a lot of solutions but impossible to fix. If u know how to fix can u show me how. Im appreciated for your help. Thanks

Added a reinstall winget to fix 23h2 installs. Click that on the config tab to fix.
